cypress/integration/home_page_spec.js

```diff
@@ -21,5 +27,39 @@ describe('The Login Page', function () {
 
     // UI should reflect this user being logged in
     cy.get('#players').should('contain', 'jane')
-  })
+    })
+})
+describe('Username already exists', function() {
+    afterEach(() => {
+        cy.request({
+            url: '/api/end',
+            method: 'GET'
+        });
+        cy.visit('/')
+        })
+    it('returns error', function () {
+        const username = 'Jane'
+        cy.request('POST', '/auth/login', { username: 'Jane' })
+        cy.get('input').type(username)
+        cy.get('button').click()
+        cy.get('h3').should('contain', 'Username already exists')
+    })
 })
+// TO BE COMPLETED IN A TIC
```

Comment:
What's wrong with this?

Reply:
needs the whole thing of making another browser so that you have someone else starting the game so that then you can be rejected when u try and join the game